lesson_03/homework_03.py

```diff
@@ -0,0 +1,218 @@
+"""The 3rd lection hometask."""
+
+"""Task 1 + code
+alice_in_wonderland = '"Would you tell me,
+please, which way I ought to go from here?"\n"
+That depends a good deal on where you want to get to," said the Cat.\n
+"I don't much care where ——" said Alice.\n
+"Then it doesn't matter which way you go," said the Cat.\n
+"—— so long as I get somewhere," Alice added as an explanation.\n
+"Oh, you're sure to do that," said the Cat, "if you only walk long enough."'
+# task 01 == Розділіть змінну alice_in_wonderland так,
+            щоб вона займала декілька фізичних лінії """
+
+alice_in_wonderland = ('"Would you tell me, please, which '
+                       'way I ought to go from here?"\n'
+                       '"That depends a good deal on '
+                       'where you want to get to," said the Cat.\n'
+                       '"I don\'t much care where ——" said Alice.\n'
+                       '"Then it doesn\'t matter '
+                       'which way you go," said the Cat.\n'
+                       '—— so long as I get somewhere," '
+                       'Alice added as an explanation.\n'
+                       '"Oh, you\'re sure to do that," '
+                       'said the Cat, "if you only walk long enough."')
+
+print(alice_in_wonderland)
+
+
+""" Task 2 + code
+# task02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті"""
+
+alice_in_wonderland = ('"Would you tell me, please, '
+                       'which way I ought to go from here?"\n'
+                       '"That depends a good deal on '
+                       'where you want to get to," said the Cat.\n'
+                       '"I don\'t much care where ——" '
+                       'said Alice.\n'
+                       '"Then it doesn\'t matter which '
+                       'way you go," said the Cat.\n'
+                       '—— so long as I get somewhere," '
+                       'Alice added as an explanation.\n'
+                       '"Oh, you\'re sure to do that," '
+                       'said the Cat, "if you only walk long enough."')
+if alice_in_wonderland.count("'") >= 2:
+    print('There are', alice_in_wonderland.count("'"), 'single quotes present')
+elif alice_in_wonderland.count("'") == 1:
+    print('There is', alice_in_wonderland.count("'"), 'single quote present')
+else:
+    print('Single quotes are missing in the text.')
```

Comment:
```suggestion
print(f' Single quotes num: {alice_in_wonderland.count("'")}')
```
All these if-else here look as overhead - you have certain text and certain task to get count of single quotes
No sense exactly here to build aka "general" solution